### DISCUSSION TITLE
notebook/index

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#6] [Announcements]
# notebook/index

https://noughtq.github.io/notebook/index.html

**Comment by tfeathers0** [comment]
请教一下这样的笔记是如何制作的，需要学习什么

**Reply by NoughtQ** [reply]
- 先在Google上搜 Mkdocs Material 教程入门
- 然后参考一下[官方的文档](https://squidfunk.github.io/mkdocs-material/getting-started/)
- 如果对语法大致了解的话，可以再学习其他大佬的笔记的源码，对自己的笔记进行个性化设计，比如[xg的笔记](https://note.tonycrane.cc/)、[修佬的笔记](https://note.isshikih.top/)等。当然前提是最好掌握Web前端技术栈（HTML、CSS、JS等）

我当初就是这么一步步走到现在的，其实我的笔记还没有完全搭好，有待美化。希望这能够帮到你 :smile:

**Reply by tfeathers0** [reply]
多谢🙏
